network-api/networkapi/buyersguide/models.py

```diff
@@ -108,7 +113,7 @@ class Meta:
         verbose_name_plural = "Buyers Guide Product Categories"
 
 
-class Product(models.Model):
+class Product(ClusterableModel):
```

Comment:
we need this type rather than `models.Model` in order to use the same `ParentalKey` concept that wagtail relies on for its InlinePanel. Without it, InlinePanels will only let you pick a single foreign relation, rather than setting as many as you want.